splink/accuracy.py

```diff
@@ -123,9 +123,17 @@ def truth_space_table_from_labels_with_predictions_sqls(
         cast(TN as float)/N as tn_rate,
         cast(FP as float)/N as fp_rate,
         cast(FN as float)/P as fn_rate,
-        cast(TP as float)/(TP+FP) as precision,
-        cast(TP as float)/(TP+FN) as recall,
-        cast(TP as float)/(TP + (FP + FN)/2) as f1
+        cast(TP as float)/(TP+FP) as precision, -- (positive predictive value)
+        cast(TP as float)/P as recall, -- SAME AS TPR (sensitivity)
+        cast(TN as float)/N as specificity, -- SAME AS TNR (selectivity)
+        cast(TN as float)/(TN+FN) as npv, -- (negative predictive value)
+        cast(TP+TN as float)/(P+N) as accuracy,
+        (2 * precision * recall)/(precision + recall) as f1,
+        (5 * precision * recall)/(4 * precision + recall) as f2,
+        (1.25 * precision * recall)/(0.25 * precision + recall) as f0_5,
+        4/((1./precision)+(1./recall)+(1./specificity)+(1./npv)) as p4,
+        cast((TP*TN)-(FP*FN) as float)/sqrt((TP+FP)*P*N*(TN+FN)) as phi
```

Comment:
We don't technically need to compute these if the user doesn't request them.

Having said that, I wouldn't remove the calculations as this is being cached.

Reply:
Yeah I figured the cost of the extra unnecessary calculation was worth it for avoiding the complexity of picking and choosing. Plus we might fancy putting some of these in the tooltips for the other charts (which is the only place several of the existing columns are used)